src/webgpu/api/validation/createBindGroup.spec.ts

```diff
@@ -312,3 +312,46 @@ TODO(#234): disallow zero-sized bindings`
       });
     }
   });
+
+g.test('minBufferBindingSize')
+  .desc('Tests that minBufferBindingSize is correctly enforced.')
+  .subcases(() =>
+    params()
+      .combine(poptions('size', [4, 256]))
+      .combine(poptions('minBindingSize', [undefined, 4, 256]))
```

Comment:
Can we also have values that are just at the limit for `size`? (like `.expand({minBufferSize} => poptions("size", [... , minBufferSize - 1]))`)?